packages/blockchain/test/iterator.spec.ts

```diff
@@ -0,0 +1,211 @@
+import * as tape from 'tape'
+
+import { Blockchain } from '../src'
+
+import { createTestDB, generateBlockchain, generateConsecutiveBlock } from './util'
+
+import type { Block } from '@ethereumjs/block'
+
+tape('blockchain test', (t) => {
+  t.test('should iterate through 24 blocks without reorg', async (st) => {
```

Comment:
Ah, great to have these tests extracted! 👍

Reply:
:slightly_smiling_face:  also enhanced the tests with reorg creation and checking

Reply:
Ah sorry, didn't have such a close look and wasn't totally obvious through this extraction and the diff therefore not displaying, but, yes: that's great too for sure! 🙂